docs/sources/setup-grafana/configure-security/configure-authentication/ldap/index.md

```diff
@@ -194,6 +194,9 @@ In `[[servers.group_mappings]]` you can map an LDAP group to a Grafana organizat
 the authoritative source. So, if you change a user's role in the Grafana Org. Users page, this change will be reset the next time the user logs in. If you
 change the LDAP groups of a user, the change will take effect the next time the user logs in.
 
+> **Note:** As of Grafana 9.5, we have introduced a feature toggle `onlyExternalOrgRoleSync` disabling the ability for synced users to change their organizational roles during a login session.
```

Comment:
We'll want to remove any mention of a specific version of Grafana. That's best practice because it leads to content tech debt. If it's available in v9.5, we can just document it. And this is a good item to add to the v9.5 what's new doc as a way to make users aware and promote its use.